### PR TITLE
Manual compression compress orderby

### PR DESCRIPTION
**Pull request by intermittentnrg**
To avoid steep performance penalty documented in GH issue: https://github.com/timescale/timescaledb/issues/5612

To be resolved by: https://github.com/timescale/timescaledb/issues/5634

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/about/latest/contribute-to-docs/)

# Review checklists

Reviewers: use this section to ensure you have checked everything before approving this PR:

## Subject matter expert (SME) review checklist

*   [ ] Is the content technically accurate?
*   [ ] Is the content complete?
*   [ ] Is the content presented in a logical order?
*   [ ] Does the content use appropriate names for features and products?
*   [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

*   [x] Is the content free from typos?
*   [x] Does the content use plain English?
*   [x] Does the content contain clear sections for concepts, tasks, and references?
*   [x] Have any images been uploaded to the correct location, and are resolvable?
*   [x] If the page index was updated, are redirects required
      and have they been implemented?
*   [x] Have you checked the built version of this content?

### CODE REVIEW

**Review by jonatas — APPROVED** [review]
Thanks for bringing the fix  @intermittentnrg !

**Review by contentana — APPROVED** [review]

**Comment by contentana** [comment]
@jonatas, I tried to fix a linting/prose issue by explaining an acronym before we used it in the code. I hope this works.

File: use-timescale/compression/manual-compression.md [hunk 1]
```diff
@@ -100,11 +100,20 @@ uncompressed chunks in your data, you can roll them up into a single compressed
 chunk.
 
 To roll up your uncompressed chunks into a compressed chunk, alter the compression
-settings to set the compress chunk time interval, and run compression operations
+settings to set the compress chunk time interval and run compression operations
 to roll up the chunks while compressing.
 
+<Highlight type="note">
+The default setting of `compress_orderby` is `'time ASC'` (the ascending or ASC command is used to sort the data returned in ascending order), which causes chunks to be re-compressed
```
Comment (billy-the-fish) [inline use-timescale/compression/manual-compression.md]: The default setting for `compress_orderby` is `time ASC`. You use `ASC` to sort the data returned in ascending order. This causes chunks to be re-compressed

File: use-timescale/compression/manual-compression.md [hunk 2]
```diff
@@ -100,11 +100,20 @@ uncompressed chunks in your data, you can roll them up into a single compressed
...
+many times during the rollup, possibly leading to a steep performance penalty. 
+Set `timescaledb.compress_orderby = 'time ASC'` to avoid this penalty.
+</Highlight>
+
+
+
 ```sql
-ALTER TABLE example SET (timescaledb.compress_chunk_time_interval = '<time_interval>');
+ALTER TABLE example SET (timescaledb.compress_chunk_time_interval = '<time_interval>',
+                         timescaledb.compress_orderby = 'time ASC');
```
Comment (billy-the-fish) [inline use-timescale/compression/manual-compression.md]: Hmmmm, weird, you are in a code block so the linter should not do this. Try indenting the first line to see if this clears the issue.

**Review by billy-the-fish — CHANGES_REQUESTED** [review]
Hi @jonatas, with a bit of luck the changes I suggested should clear the linter errors.

File: use-timescale/compression/manual-compression.md (hunk 1 above)
Comment (intermittentnrg) [reply use-timescale/compression/manual-compression.md]: The default settings it `time DESC`. Changing to ASC avoids re-compression https://github.com/timescale/timescaledb/issues/5612#issuecomment-1525741498

Sorry, my bad writing..

File: use-timescale/compression/manual-compression.md (hunk 2 above)
Comment (intermittentnrg) [reply use-timescale/compression/manual-compression.md]: Fixed, pending approval of workflow to re-run.

File: use-timescale/compression/manual-compression.md (hunk 1 above)
Comment (intermittentnrg) [reply use-timescale/compression/manual-compression.md]: Not sure what you're asking? Do you want to replace the text with above?

**Review by billy-the-fish — APPROVED** [review]

File: use-timescale/compression/manual-compression.md (hunk 1 above)
Comment (billy-the-fish) [reply use-timescale/compression/manual-compression.md]: Your fix is awesome, thank you.

**Pull request merged**